src/styles/main.scss

```diff
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
```

Comment:
Consider defining a SCSS variable for the width value `706px` to improve maintainability and readability. This will make it easier to update the value in the future if needed.